starter-code/css/main.css

```diff
@@ -6,3 +6,85 @@ Green: #00B172
 White: #FFF
 
 */
+body{
+  margin: 0;
+  font-family: sans-serif;
+}
+nav{
```

Comment:
try using the box-sizing: border-box; property to get the format right on this one. Border box allows us set margin and padding and have it take away from a set width rather then add a set width.

`
.nav {
    overflow: auto;
    display: block;
    position: fixed;
    top: 0;
    background-color: white;
    width: 100%;
    height: 40px;
    padding: 5px 112px 0;
    box-sizing: border-box;
}

This will give you a clean layout, almost spot on with the original

`